openrtb2/audio.go

```diff
@@ -38,40 +38,112 @@ type Audio struct {
 	//   Maximum audio ad duration in seconds.
 	MaxDuration int64 `json:"maxduration,omitempty"`
 
+	// Attribute:
+	//   poddur
+	// Type:
+	//   integer; recommended
+	// Description:
+	//   Indicates the total amount of time that advertisers may fill for a
+	//   "dynamic" audio ad pod, or the dynamic portion of a "hybrid"
+	//   ad pod. This field is required only for the dynamic portion(s) of
+	//   audio ad pods. This field refers to the length of the entire ad
+	//   break, whereas minduration/maxduration/rqddurs are
+	//   constraints relating to the slots that make up the pod.
+	PodDur int64 `json:"poddur,omitempty"`
+
 	// Attribute:
 	//   protocols
 	// Type:
 	//   integer array; recommended
 	// Description:
-	//   Array of supported audio protocols. Refer to List 5.8.
-	Protocols []Protocol `json:"protocols,omitempty"`
+	//   Array of supported audio protocols. Refer to List: Creative
+	//   Subtypes - Audio/Video in AdCOM 1.0.
+	Protocols []int8 `json:"protocols,omitempty"`
```

Comment:
All enumeration lists (formerly section 5) have been removed from OpenRTB 2.6 in favor of the definitions in AdCOM 1.0. I disassociated the enums for now, but do you think it's better to directly reference the /adcom1 package here? .. or copy the AdCOM enums to /openrtb2?

Reply:
Just commented above, I think it will be better to import adcom1 in openrtb2 since it relies so heavily on adcom now.

I'll get back to it once done with other bits.